Pages/basePage.js

```diff
@@ -0,0 +1,64 @@
+import { By, until } from 'selenium-webdriver';
+import { DEFAULT_TIMEOUT, DEFAULT_MIN_WAIT_TIME } from '../Utils/helpers/constants.js';
+
+export default class basePage {
+  constructor(driver, logger = null) {
+    this.driver = driver;
+    this.logger = logger || { info: () => {} };
+  }
+
+  setLogger(logger) {
+    this.logger = logger && typeof logger.info === 'function' ? logger : { info: () => {} };
+  }
+
+  async open() {
+    this.logger.info('Base: open https://nodejs.org');
+    await this.driver.get('https://nodejs.org');
+  }
```

Comment:
У тебя метод `open` реализован в двух местах. Здесь и в классе `NodejsHomePage`, при этом `NodejsHomePage` наследует `BaseNodePage`.
При этом они у тебя практически идентичные с минимальным отличием в линке. То есть ты переписываешь родительский метод.

Я бы предложил изменить текущий метод следующим образом.
Назвать его `openPage` и он должен принимать линку.
```suggestion
  async openPage(url) {
    this.logger.info(`Base: open ${url}`);
    await this.driver.get(url);
  }
```
А в классе `NodejsHomePage` в методе `open` использовать в этот родительский метод `openPage` и просто передавать линку домашней страницы.
Т.е. вот так
```javascript
  async open() {
    await this.openPage('https://nodejs.org/en');
  }
```

Саму же линку на домашнюю страницу, по-хорошему, надо в константы закинуть.